packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -728,6 +732,57 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		return { age: revertAge };
 	}
 
+	/**
+	 * Recursively creates a revertible for the given commit and its ancestors.
+	 */
+	private forkRevertible(
+		view: TreeCheckout,
+		commit: GraphCommit<SharedTreeChange>,
+		data: CommitMetadata,
+	): Revertible {
+		const forkRevertibleFunction = this.forkRevertible.bind(this);
```

Comment:
You should be able to rewrite this using an arrow function.
```typescript
const` forkRevertibleFunction = (newView: TreeCheckout) => this.forkRevertible(newView.fork(), commit, data);
```

This should simplify your code because you no longer need to worry about losing the this context when passing functions around.